app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class FeedbacksController < ApplicationController
+  before_action -> { authorize! @feedback, with: FeedbackPolicy }
+
+  def new
+    @feedback = Feedback.new
+  end
+
+  def create
+    result = Feedbacks::Action.new.call(feedback_params)
+
+    if result.success?
+      flash.now[:notice] = "Feedback sent successfully"
+      render :thanks
+    else
+      flash.now[:alert] = result.failure[:message]
```

Comment:
When I try to send an empty feedback:
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/35406008/233769245-d13c8ed5-98d4-4d55-b53a-041518b39a88.png">